src/inversion_scripts/operators/TROPOMI_operator.py

```diff
@@ -257,13 +257,13 @@ def apply_average_tropomi_operator(
             perturbations = np.full(n_elements, 1.0, dtype=float)
 
             if config["OptimizeOH"]:
-                oh_perturbation = config["PerturbValueOH"]
+                oh_perturbation = float(config["PerturbValueOH"]) - 1.0
             else:
-                oh_perturbation = 1.0
+                oh_perturbation = 1.0 # should these be 0.0?
             if config["OptimizeBCs"]:
                 bc_perturbation = config["PerturbValueBCs"]
             else:
-                bc_perturbation = 1.0
+                bc_perturbation = 1.0 # should these be 0.0?
```

Comment:
@eastjames can you advise on whether these should be 1.0 or 0.0? I imagine 0.0 could give some divide by zero errors.

Reply:
I think this is 1.0 because these are the cases where you're not optimizing OH or BCs so there's no subtracting 1 involved